app/controllers/idv/jurisdiction_controller.rb

```diff
@@ -44,5 +44,9 @@ def jurisdiction_params
     def set_jurisdiction_form
       @jurisdiction_form ||= Idv::JurisdictionForm.new
     end
+
+    def failure_url(reason)
+      idv_jurisdiction_failure_url(reason)
```

Comment:
Doesn't look like we have cases that need a `failure_url`?

Reply:
This is used by a few of the IdV concerns:

- [IdvSessionConcern](https://github.com/18F/identity-idp/blob/800c5e7863b10dfb77ed4dbdb725e57331138dde/app/controllers/concerns/idv_session.rb#L12)
- -[IdvStepConcern](https://github.com/18F/identity-idp/blob/800c5e7863b10dfb77ed4dbdb725e57331138dde/app/controllers/concerns/idv_step_concern.rb#L31)